dev/src/platform.cpp

```diff
@@ -279,9 +284,10 @@ bool InitPlatform(string _maindir, string_view auxfilepath, bool from_bundle,
     return true;
 }
 
-void AddDataDir(string_view path, bool is_rel) {
-    auto fpath = SanitizePath(path);
-    if (is_rel) fpath = projectdir + fpath;
+void AddDataDir(string_view path) {
+    // If the import is written like --import /project/abc add slash at the end.
```

Comment:
shouldn't you check if there already is a `FILESEP` at the end?

Reply:
If I remember correctly, its intentional design in file paths that double slashes have no meaning (Atlough I haven't seen it on black and white so take it with a grain of salt).
So D:\Project\\test.lobster is valid and for linux /project//test.lobster is also valid.

Reply:
I know, but just because it is legal doesn't mean we don't want to generate nice looking paths :)